cmd/argo/commands/list.go

```diff
@@ -64,6 +64,34 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows",
+		Example: `# List all workflows:
+  argo list
+
+# Show workflows from all namespaces:
+  argo list -A
+
+# List all completed worflows:
+  argo list --completed
+
+# List completed workflows finished before 2h:
```

Comment:
```suggestion
# List workflows that finished more than 2h ago:
```

- "completed" + "finished" are redundant together
- some grammar fixes
- we could be more explicit than "before", which is a bit more difficult to understand (since one has to interpret the directionality). "more than" has a direction specified (3h is more than 2h etc)